nucypher/blockchain/eth/actors.py

```diff
@@ -1202,6 +1206,26 @@ def missing_commitments(self) -> int:
         missing = self.staking_agent.get_missing_commitments(checksum_address=staker_address)
         return missing
 
+    def verify_policy_payment(self, hrac: bytes) -> None:
+        arrangements = self.policy_agent.fetch_policy_arrangements(policy_id=hrac)
+        members = set()
+        for arrangement in arrangements:
+            members.add(arrangement.node)
+            if self.checksum_address == arrangement.node:
+                return
+        else:
+            if not members:
+                raise Policy.Unknown(f'{hrac.hex()} is not a published policy.')
+            raise Policy.Unpaid(f"{hrac.hex()} is unpaid.")
+
+    def verify_active_policy(self, hrac: bytes) -> None:
+        policy = self.policy_agent.fetch_policy(policy_id=hrac)
+        if policy.disabled:
+            raise Policy.Inactive(f'{hrac.hex()} is a disabled policy.')
+        expired = datetime.utcnow() >= datetime.utcfromtimestamp(policy.end_timestamp)
+        if expired:
```

Comment:
Does this mean that if a policy is disabled, and then subsequently the expiration date passes, we'll get a different error?  Is that what we want?